sections/06_discussion.md

```diff
@@ -68,46 +72,46 @@ interpretable healthcare features from trained deep models.
-level features?  Is there still work to be done in finding the right low-level
-features in some problems?*
-
-###### Biomedical data is often "Wide"
-
-*Biomedical studies typically deal with relatively small sample sizes but each
-sample may have millions of measurements (genotypes and other omics data, lab
-tests etc).*
-
-*Classical machine learning recommendations were to have 10x samples per number
-of parameters in the model.*
-
-*Number of parameters in an MLP. Convolutions and similar strategies help but do
-not solve*
-
-*Bengio diet networks paper*
-
+A lack of large-scale, high-quality, labeled training data has impacted deep
+learning in nearly all applications we have discussed, from healthcare to
+genomics to drug discovery.  The challenges of training complex, high-parameter
+neural networks from few examples are obvious, but uncertainty in the labels of
+training instances can be just as problematic.  In genomics, for example,
+labeled data may be derived from an experimental assay that has known and
+unknown technical artifacts, biases, and error profiles.  It is possible to
+weight training examples or construct Bayesian models to account for uncertainty
+in the data, but this is rarely done in practice. `TODO: Is this even true?
+Would like feedback here.`
+
+For some types of data, especially images, it is straightforward to augment
+training datasets by splitting one labeled example into multiple examples. An
+image can easily be rotated or inverted and retain its label `TODO: does someone
+have examples to cite here? Issue #163?`.  3D MRI and 4D fMRI (with time as a
+dimension) data can be decomposed into sets of 2D images [@doi:10.1101/070441].
+This greatly expands the number of training examples in domains where only
+hundreds of subjects are available but artificially treats images from the same
+volume as independent instances.  CellCnn trains a model to recognize rare cell
+populations in single-cell data by creating training instances that consist of
+random subsets of cells that are randomly sampled with replacement from the
+full dataset [@tag:Arvaniti2016_rare_subsets].
+
+Simulated or semi-synthetic training data has also been employed in multiple
+biomedical domains. `TODO:  simulated data: #5 #99 #293, maybe #117 and #197.
+There is a counter-example from drug discovery to include as well
+that is related to #55`
+
+Multimodal, multi-task, and transfer learning, discussed in detail below, can
+also combat data limitations to some degree.  There are also emerging
+network architectures, such as Diet Networks for high-dimensional SNP data
+[@tag:Romero2017_diet], that drastically reduce the number of free parameters
+by predicting parameters (weights) as a function of each SNP's embedding
+in some feature space instead of directly optimizing a weight for each SNP.
+`TODO: not very familiar with this paper, can someone describe it better?`
```

Comment:
@brettbj thanks, I added this.  I want to rephrase the `x2vec style` or remove the example embedding types and will use everything else as-is.